js/classes/Helper.js

```diff
@@ -350,6 +350,17 @@
 		}
 	};
 
+	/**
+	 * A function that takes care of escaping @ character which could be interpreted
+	 * as CSS notation. This is due to the fact that jQuery uses CSS syntax for selecting elements.
+	 * These characters must be escaped by placing two backslashes in front of them.
+	 * @param {string} selector jQuery element selector
```

Comment:
This function should turn an ID into a selector, correct? Wouldn't e.g. `elementId` be a better parameter name than `selector`?